solver/jobs.go

```diff
@@ -455,7 +455,7 @@ func (jl *Solver) NewJob(id string) (*Job, error) {
 }
 
 func (jl *Solver) Get(id string) (*Job, error) {
-	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 6*time.Second)
```

Comment:
@tonistiigi this is to be above the 5 seconds timeout of the `sessionManager.Any()` ?

Reply:
These shouldn't be related. One should never wait for another.